src/hooks/useOptions.ts

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import { ethers } from "ethers";
+import { useClaims } from "./useClaims";
+import { useConnection } from "./useConnection";
+import { contracts } from "../config";
+import { isValidChainId } from "../utils/chainId";
+import { useTransactions } from "./useTransactions";
+
+export function useOptions() {
```

Comment:
i think we talked about this useOptions before, its a really ambiguous name, maybe like `useKpiOptions` or a more specific name somehow